### DISCUSSION TITLE
Screen alignment

### DISCUSSION THREAD

**Discussion by A-Winters** [SubtitleEdit/subtitleedit#4920] [General]
I've opened an .ass file for editing and the subtitles won't position correctly. I've tried to align them and it seems that up is down, left is right, middle or off screen - no consistency.

The code at the top says the file was initially made on Aegisub.
I don't know if they are affecting the alignment issue, what to change or what's safe to delete.

How can I make this video work with Subtitle Edit, please?

**Comment by niksedk** [comment]
Screenshots + subtitles (as .zip) can be attached and both helps a lot :)

You can right-click on the list view and choose alignment (or use shortcuts):
![image](https://user-images.githubusercontent.com/1261331/112098224-bbb30800-8ba1-11eb-988a-f8203ba4855e.png)

Alignment values are based on the numeric keypad layout. {\an1} - bottom left, {\an2} - bottom center, {\an3} - bottom right, {\an4} - center left, {\an5} - center center, {\an6} - center right, {\an7} - top left, {\an8} - top center, {\an9} - top right.

**Comment by OmrSi** [comment]
First of all, you should use MPV as your video player in SE.
Secondly, here is how it works:

![image](https://user-images.githubusercontent.com/20923700/112149380-9648ed00-8be7-11eb-8f7b-2d9b8f51d047.png)

**Comment by A-Winters** [comment]
Thank you both! It's helped me to understand how alignment adjustments work.
I see now that the problem is that my top left corner doesn't start at 0. It's hard to pinpoint exactly what number it starts at (close to 100, I think).
The centre of the video is 200,200 (if that helps).
Some of the dialogue was positioned way out of sight, which I adjusted, but I expect that I'm making matters worse as the screen size is all wrong.

I can't work out if there's a way to fix it.

**Reply by OmrSi** [reply]
It depends on the Ass file properties, the file might be made for a certain resolution and you might be using it on another one.
